backend/clickhouse/metrics.go

```diff
@@ -69,8 +279,28 @@ func (client *Client) ReadEventMetrics(ctx context.Context, projectID int, param
 	})
 }
 
+func (client *Client) QuerySessionCustomMetrics(ctx context.Context, projectId int, sessionSecureId string, created time.Time, metricNames []string) (*modelInputs.MetricsBuckets, error) {
+	day := 24 * time.Hour
+	query := strings.Join(lo.Map(metricNames, func(m string, _ int) string {
+		return fmt.Sprintf("%s=%s", modelInputs.ReservedMetricKeyMetricName.String(), m)
+	}), " OR ")
+	query = fmt.Sprintf("%s=%s AND %s", modelInputs.ReservedMetricKeySecureSessionID.String(), sessionSecureId, query)
```

Comment:
does this need to be parenthesized since `query` can include `OR` expressions? `%s=%s AND (%s)`

Reply:
good catch